### PR TITLE
Additions to integration tests for recent changes.

### PR DESCRIPTION
**Pull request by alxdarksage**

### CODE REVIEW

File: src/test/java/org/sagebionetworks/bridge/sdk/integration/CRCTest.java [hunk 1]
```diff
@@ -197,6 +195,21 @@ public void createAppointment() throws IOException {
         StudyParticipant participant = adminUser.getClient(ParticipantsApi.class)
                 .getParticipantById(user.getUserId(), false).execute().body();
         assertTrue(participant.getDataGroups().contains("tests_scheduled"));
+        
+        HealthDataRecordList records = user.getClient(InternalApi.class)
```
Comment (DwayneJengSage) [inline src/test/java/org/sagebionetworks/bridge/sdk/integration/CRCTest.java:199]: This uses a global secondary index. Consider adding a 1-2 second sleep.

Note that we have been having a lot of Global Secondary Index failures lately. See https://sagebionetworks.jira.com/browse/BRIDGE-2828  (which I think would be a good task for an intern.)

File: src/test/java/org/sagebionetworks/bridge/sdk/integration/CRCTest.java [hunk 2]
```diff
@@ -197,6 +195,21 @@ public void createAppointment() throws IOException {
...
+                .getHealthDataByCreatedOn(DateTime.now().minusMinutes(10), DateTime.now().plusMinutes(10))
```
Comment (DwayneJengSage) [inline src/test/java/org/sagebionetworks/bridge/sdk/integration/CRCTest.java:200]: Also, clock skew on Jenkins has been known to exceed more than 10 minutes. I recommend setting this to plus/minus an hour. (If clock skew gets that high, we probably have other concerns.)

**Review by DwayneJengSage — CHANGES_REQUESTED** [review]

File: src/test/java/org/sagebionetworks/bridge/sdk/integration/CRCTest.java (hunk 1 above)
Comment (alxdarksage) [reply src/test/java/org/sagebionetworks/bridge/sdk/integration/CRCTest.java:199]: Added

File: src/test/java/org/sagebionetworks/bridge/sdk/integration/CRCTest.java [hunk 3]
```diff
@@ -271,10 +259,17 @@ public void createProcedureRequest() throws IOException {
         StudyParticipant participant = adminUser.getClient(ParticipantsApi.class)
                 .getParticipantById(user.getUserId(), false).execute().body();
         assertTrue(participant.getDataGroups().contains("tests_collected"));
+        
+        Thread.sleep(2000);
+
+        HealthDataRecordList records = user.getClient(InternalApi.class)
+                .getHealthDataByCreatedOn(DateTime.now().minusMinutes(10), DateTime.now().plusMinutes(10))
```
Comment (DwayneJengSage) [inline src/test/java/org/sagebionetworks/bridge/sdk/integration/CRCTest.java:266]: Should also change the time range here.

File: src/test/java/org/sagebionetworks/bridge/sdk/integration/CRCTest.java [hunk 4]
```diff
@@ -317,6 +312,12 @@ public void createObservation() throws IOException {
         StudyParticipant participant = adminUser.getClient(ParticipantsApi.class)
                 .getParticipantById(user.getUserId(), false).execute().body();
         assertTrue(participant.getDataGroups().contains("tests_available"));
+        
+        Thread.sleep(2000);
+        
+        HealthDataRecordList records = user.getClient(InternalApi.class)
+                .getHealthDataByCreatedOn(DateTime.now().minusMinutes(10), DateTime.now().plusMinutes(10))
```
Comment (DwayneJengSage) [inline src/test/java/org/sagebionetworks/bridge/sdk/integration/CRCTest.java:319]: Should also change the time range here.

**Review by DwayneJengSage — CHANGES_REQUESTED** [review]

**Review by DwayneJengSage — APPROVED** [review]

**Pull request merged**